docs/firestore/usage.mdx

```diff
@@ -195,6 +195,7 @@ FirebaseFirestore.instance
         querySnapshot.docs.forEach((doc) {
             print(doc["first_name"]);
         });
+        //
```

Comment:
Why add an empty comment here?

Reply:
This was a  unintended missplacment!
But, if I may, can it be a cause for the CI rejection?